WebSuggest/websuggest.py

```diff
@@ -307,11 +309,18 @@ def on_suggest(self, user_input, items_chain):
             self.warn('Item definition not found in current config: "{}"'.format(profile_name))
             return
 
+        default_item = current_item.clone()
+        default_item.set_args(user_input)
+        if not user_input:
+            default_item.set_short_desc("Open the search engine home page")
+
+        suggestions = [default_item]
+
         # avoid doing unnecessary network requests in case user is still typing
-        if len(user_input) < 2 or self.should_terminate(0.25):
+        if len(user_input) < 2 or self.should_terminate(self.waiting_time):
+            self.set_suggestions(suggestions)

```

Comment:
This is for the case of entering one character. But I agree that this is unnecessary. Nobody is looking for 1 character.